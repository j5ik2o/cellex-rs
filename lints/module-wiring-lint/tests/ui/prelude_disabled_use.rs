#![feature(register_tool)]
#![register_tool(module_wiring)]
#![warn(module_wiring::no_parent_reexport)]

mod prelude {
  pub struct X;
}

pub use prelude::X;

fn main() {}
