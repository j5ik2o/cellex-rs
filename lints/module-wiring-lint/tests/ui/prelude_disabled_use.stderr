warning: 再エクスポートは末端モジュールの直属親以外では禁止です
  --> $DIR/prelude_disabled_use.rs:9:1
   |
LL | pub use prelude::X;
   | ^^^^^^^^^^^^^^^^^^^
   |
   = note: ルール: 原則ルール: 再エクスポート禁止
   = note: 詳細: prelude モジュールからの再エクスポートは無効化されています
   = help: プレリュードではなく対象モジュールの直属親から公開してください
   = note: `#[warn(no_parent_reexport)]` on by default

warning: 1 warning emitted

